src/routes/spotify/desktop/Attract.svelte

```diff
@@ -18,112 +17,51 @@
     visible: boolean;
   } = $props();
 
-  const defaultMessages = [
-    "Welcome to JukeLab",
-    "Press any key to continue",
-    "Tap anywhere to start",
-    "Select your favorite tunes",
-  ];
-
-  const holidayMessages = [
-    "Happy Holidays!",
-    "Tap anywhere for turkey",
-    "Select your favorite holiday tunes",
-  ];
-
-  const messages = $derived($holiday ? holidayMessages : defaultMessages);
-
-  let message = $state(messages[0]);
-  let artTop = $state<AlbumTracks[]>([]);
-  let artBottom = $state<AlbumTracks[]>([]);
-
-  const reshuffle = () => {
-    const shuffled = shuffle([...playlist.albums]);
-    const chunks = chunk(shuffled, Math.ceil(shuffled.length / 2));
-    artTop = chunks[0];
-    artBottom = chunks[1];
-  };
-
-  onMount(() => {
-    reshuffle();
-
-    if (browser) {
-      setInterval(() => {
-        const i = messages.findIndex((m) => m == message);
-        message = messages[(i + 1) % messages.length];
-      }, 5000);
-    }
-  });
-
-  $effect(() => {
-    // reshuffle while invisible
-    visible || reshuffle();
-  });
+  let format = $state("default");
+  const onClose = () => (visible = false);
 </script>
 
 {#if visible}
-  {@const {
-    album: { art },
-    track: { title, artist },
-  } = playlist}
-
-  <button
-    class="flex h-screen w-screen flex-col items-center justify-center border bg-slate-200 pb-20"
+  <div
+    role="button"
+    aria-label="Close attract screen"
+    tabindex="0"
+    class="flex h-screen w-screen flex-col items-center justify-center bg-slate-200"
     in:fade
     out:fade
-    onclick={() => {
-      visible = false;
-    }}
+    onclick={onClose}
+    onkeydown={onClose}
```

Comment:
started down this path to avoid the button inside a button problem

but now that this is a div, svelte asks for a key handler together with the onclick